cpp/test/sparse/masked_matmul.cu

```diff
@@ -87,7 +87,8 @@ bool isCuSparseVersionGreaterThan_12_0_1()
 template <typename value_t,
           typename output_t,
           typename index_t,
-          typename bitmap_t      = uint32_t,
+          bool bitmap_or_bitset  = true,
```

Comment:
Is this expecting `true` when it's one or the other? This is a little confusing. Please make an enum for this so that it's more clear to future eyes who might need to update this code.